scripts/source_coverage.sh

```diff
@@ -30,18 +25,25 @@ tari_merge_mining_proxy
 tari_metrics
 tari_miner
 tari_mining_helper_ffi
-tari_p2p
-tari_service_framework
 tari_test_utils
 tari_wallet_ffi
```

Comment:
This actually has some good tests in it, and would be good to track the coverage